ksqldb-serde/src/main/java/io/confluent/ksql/serde/json/JsonFormat.java

```diff
@@ -55,6 +55,11 @@ protected ConnectSchemaTranslator getConnectSchemaTranslator(
     throw new UnsupportedOperationException(name() + " does not implement Schema Registry support");
   }
 
+  @Override
+  public Set<String> getSupportedProperties() {
+    return JsonProperties.SUPPORTED_PROPERTIES;
```

Comment:
Why this change? I though JSON won't support this, only JSON_SR ?

Might be easier to just `return ImmutableSet.of();` ?

Reply:
Because I introduced `JsonProperties`, so make it consistent with other format properties class to have `SUPPORTED_PROPERTIES` field. So we can modify `SUPPORTED_PROPERTIES` when new properties are added later.